packages/twenty-front/vite.config.ts

```diff
@@ -133,7 +133,7 @@ export default defineConfig(({ command, mode }) => {
     ],
 
     optimizeDeps: {
-      exclude: ['node_modules/.vite', 'node_modules/.cache'],
+      exclude: ['../../node_modules/.vite', '../../node_modules/.cache'],
```

Comment:
This was from a previous PR I raised and I fixed it because I think I had but an incorrect path 🤔

Reply:
Sorry for hijacking the PR :)